index.js

```diff
@@ -2198,3 +2212,29 @@ function incrementNodeInspectorPort(args) {
     return arg;
   });
 }
+
+function parseArg(arg) {
```

Comment:
I was going to suggest moving this, but changed my mind when I looked deeper, so leave it here for now. I'm adding a comment partly to remind myself! 

In general I now prefer adding internal class-related code as a private method on the class (weakly private) rather than a separate private function (strongly private).

BUT I see I added `_parseOptionFlags()` as a private function. It was easier to call it from both the Option class and the Command class while refactoring. That is likely to be the case here too.

Reply:
Yep, i wanted to share the same code between the two classes. Personally I think that extracting logic into pure functions that don't contain state (this) is most of the time a good thing, but if you have a different approach I can align my code of course.